src/components/TransactionDialog/TransactionDialog.tsx

```diff
@@ -15,58 +35,183 @@ export type TransactionDialogProps = Pick<ActionDialogProps, 'className'> & {
 
 const TRANSACTION_STEPS_DETAILS = {
   [ExtrinsicStatus.ProcessingAssets]: {
-    title: 'Processing assets...',
+    title: 'Processing your assets',
     description:
       "Please wait till all your assets get processed. This can take up to 1 minute, depending on asset size and your machine's computing power.",
-    tooltip: '',
+    tooltip: 'Processing',
+    animation: {
+      data: firstStepAnimation,
+      size: {
+        width: 216,
+        height: 216,
+      },
+      loop: true,
+    },
   },
   [ExtrinsicStatus.Unsigned]: {
-    title: 'Waiting for signature...',
+    title: 'Sign in Polkadot',
     description: 'Please sign the transaction using the Polkadot browser extension.',
     tooltip: 'Signature',
+    animation: {
+      data: secondStepAnimation,
+      size: {
+        width: 288,
+        height: 216,
+      },
+      loop: false,
+    },
   },
   [ExtrinsicStatus.Signed]: {
-    title: 'Waiting for confirmation...',
+    title: 'Processing transaction',
     description:
       'Your transaction has been signed and sent. Please wait for the blockchain confirmation. This should take about 15 seconds.',
     tooltip: 'Confirmation',
+    animation: {
+      data: thirdStepAnimation,
+      size: {
+        width: 216,
+        height: 216,
+      },
+      loop: true,
+    },
   },
   [ExtrinsicStatus.Syncing]: {
-    title: 'Waiting for data propagation...',
+    title: 'Propagating changes',
     description:
       "Your transaction has been accepted and included into the blockchain. Please wait till it's picked up by the indexing node. This should take up to 15 seconds.",
     tooltip: 'Propagation',
+    animation: {
+      data: fourthStepAnimation,
+      size: {
+        width: 144,
+        height: 144,
+      },
+      loop: true,
+    },
+  },
+  [ExtrinsicStatus.Error]: {
+    title: 'Something went wrong...',
+    description:
+      'An unexpected error was encountered. If this persists, our Discord community may be a good place to find some help.',
+    tooltip: 'Propagation',
+    animation: {
+      data: errorAnimation,
+      size: {
+        width: 288,
+        height: 264,
+      },
+      loop: false,
+    },
```

Comment:
Seems the details for error were provided here but I'm still seeing the old error dialog:
<img width="515" alt="Screenshot 2021-09-22 at 16 13 53" src="https://user-images.githubusercontent.com/12646744/134360364-e2dc0f61-7257-469e-90e6-0ca75d1ae2c1.png">

Figma reference: https://www.figma.com/file/Vk2Z4QOiVa5bB6q3cBIG5J/Joystream-Atlas?node-id=9066%3A227100